### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by Siemko** [chaance/tsdx#2] [General]
Hi @chaance, thank you for picking up this package!

Do you have any roadmap for making this fork production-ready? Do you need any help with this one?

I would like to use that in the future project (starting sometime in the first half of September 2021) and it will be extremely useful to know if it will be possible.

If you need any help, please open issues and I would gladly help whenever it will be possible!

**Comment by chaance** [comment]
Hey @Siemko thanks for the interest. I haven't had time to write out my thoughts, but I do have a few. Is there anything in particular you're looking for? Issues should be open as is, but happy to use this discussion as a starting point for such a plan!

**Reply by Siemko** [reply]
I would start with replacing the bundler with `esbuild`, using `rollup-plugin-esbuild` instead of `rollup-plugin-typescript2`, `@rollup/plugin-typescript` and `rollup-plugin-terser`.
